tachio/src/remember.js

```diff
@@ -397,28 +395,27 @@ async function getChannelMessageHistory(channelId, limit = 5) {
   return data;
 }
 
-// /**
-//  * Embeds a string using the Voyage AI API.
-//  * @param {string} string - The input string to embed.
-//  * @param {string} [model="voyage-large-2"] - The model to use for embedding (default: "voyage-large-2").
-//  * @returns {Promise<object>} - A promise that resolves to the response data from the API.
-//  */
-// async function voyageEmbedding(string, model = "voyage-large-2") {
-//   const response = await axios.post(
-//     "https://api.voyageai.com/v1/embeddings",
-//     {
-//       input: string,
-//       model,
-//     },
-//     {
-//       headers: {
-//         "Content-Type": "application/json",
-//         Authorization: `Bearer ${process.env.VOYAGE_API_KEY}`,
-//       },
-//     },
-//   );
-//   return response.data;
-// }
+/**
+ * Embeds a string using the Voyage AI API.
+ * @param {string} input - The input string to embed.
+ * @param {string} [model="voyage-large-2-instruct"] - The model to use for embedding (default: "voyage-large-2-instruct").
+ * @returns {Promise<object>} - A promise that resolves to the response data from the API.
+ */
+async function voyageEmbedding(input, model = "voyage-large-2-instruct") {
+  const response = await fetch(VOYAGE_AI, {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json',
+      Authorization: `Bearer ${process.env.VOYAGE_API_KEY}`
+    },
+    body: JSON.stringify({
```

Comment:
We should also specify the `input_type` parameter for optimal retrieval. Here is the [explanation from Voyage AI](https://docs.voyageai.com/docs/embeddings#python-api):

> When the `input_type` is set to `none`, and the input text will be directly encoded by our embedding model. Alternatively, when the inputs are documents or queries, the users can specify input_type to be `query` or `document`. In such cases, Voyage will prepend a special prompt to input text and send the extended inputs to the embedding model.
> **For retrieval/search use cases, we recommend specifying this argument when encoding queries or documents to enhance retrieval quality. Embeddings generated with and without the input_type argument are compatible.**

Reply:
@kyphan1